src/app/index.js

```diff
@@ -59,6 +46,70 @@ export default yeoman.generators.Base.extend({
   },
 
   /**
+   * Gets the name of the scope including the "@" symbol.
+   *
+   * @param  {String} scope
+   * @return {String}
+   */
+  _getScope(scope) {
+    return scope && typeof scope === 'string' ? `@${scope}` : '';
```

Comment:
It's enforced in the [validation for the prompt input](https://github.com/videojs/generator-videojs-plugin/pull/43/files#diff-22e911e52dc237dc1da62b1051879718R39), but I think it would make sense to double-check (since prompts can be skipped, etc).